.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,12 @@
+## Pre-Request Checklist
```

Markdownlint (0.37.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)